sharepoint/sharepoint-ps/sharepoint-online/Set-SPORestrictedSiteCreation.md

```diff
@@ -0,0 +1,178 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-sporestrictedsitecreation
+applicable: SharePoint Online
+title: Set-SPORestrictedSiteCreation
+schema: 2.0.0
+author: vgaddam-pm
+ms.author: vgaddam
+ms.reviewer: jmcdowe
+---
+
+# Set-SPORestrictedSiteCreation
+
+## SYNOPSIS
+
+Sets or updates one or more group configurations for restricting site creation.
+
+## SYNTAX
+
+### ParamSet1
+
+```powershell
+Set-SPORestrictedSiteCreation
+   [-Enabled <Boolean>]
+   [-Mode <RestrictedSiteCreationMode>]
+   [-SiteType <RestrictedSiteCreationSiteType>]
+   [-RestrictedSiteCreationGroups <String>]
+```
+
+## DESCRIPTION
+
+The Set-SPORestrictedSiteCreation cmdlet sets or updates the configuration or setting for the Restricted Site Creation feature.
+
+You must be a SharePoint Online administrator to run the cmdlet.
+
+For permissions and the most current information about Windows PowerShell for SharePoint Online, see the online documentation at [Intro to SharePoint Online Management Shell](https://learn.microsoft.com/powershell/sharepoint/sharepoint-online/introduction-sharepoint-online-management-shell?view=sharepoint-ps).
+
+> [!Important]
+> You must use version 16.0.25513 (published November 2024) or later of the [SharePoint Online Management Shell](https://www.microsoft.com/en-us/download/details.aspx?id=35588) for these commands to function properly. Earlier versions do not have the current list of site types and will not operate correctly.
+
```

Comment:
remove this -- it's not useful. admins know that if they can't find a cmdlet they know to upgrade to a later build

Reply:
Hi Sam, this is the issue I believe Matthew Calligaro talked to you about. the Cmdlet will be available at a prior build but won't function properly because the enum mapping will change with OneDrive support

Reply:
Note that we also don't know yet what exactly the November build will be. 25513 is a good guess for now based on the fork date, but these numbers have changed in the past, e.g., by PM deciding to pick up a later build.

Reply:
Thank you for pointing this out, will keep this in mind.